testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -717,10 +743,12 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
 
     if not use_local_testserver and create_db_per_test:
         log_info("Starting TestServer...")
+        testserver.stop()
         if device_enabled:
             testserver.start_device(log_filename)
         else:
             testserver.start(log_filename)
+        time.sleep(2)
```

Comment:
Is there a way we can avoid sleep?

Reply:
The thing about sleeps is that they might change between different machines: 2 might work fro one machine but not for the other, which might cause the test to be flaky. Is there a way to query that the CBL has started, rather than wait?